lab_1_classify_by_unigrams/main.py

```diff
@@ -20,7 +19,16 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
-
+    if isinstance(text, str):
```

Comment:
revert the condition and return `None` as soon as possible

Reply:
Why?

Reply:
It is a good style - we tried it on seminars https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
done